proxy-injector/templates/servicecomb/servicecomb.go

```diff
@@ -0,0 +1,200 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one or more
+ *  contributor license agreements.  See the NOTICE file distributed with
+ *  this work for additional information regarding copyright ownership.
+ *  The ASF licenses this file to You under the Apache License, Version 2.0
+ *  (the "License"); you may not use this file except in compliance with
+ *  the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package servicecomb
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"strings"
+	"sync"
+	"text/template"
+
+	"github.com/apache/servicecomb-mesher/proxy-injector/templates"
+	"gopkg.in/yaml.v2"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	TemplateName          = "servicecomb-mesher"
+	envVersion            = "VERSION"
+	envListenPortName     = "LISTEN_PORT_NAME"
+	defaultVersion        = "1.0.0"
```

Comment:
Should these be in the configuration？

Reply:
Yes, these need to be configured in the configmap of Kubenetes  and used in the POD as environment variables.